### PR TITLE
Add Homepage front matter

### PR DESCRIPTION
**Pull request by jamesericdavidson**
Use the native `content/_index.md` file and `{{ .Content }}` variable to display front matter.

An example of the feature has been added to `exampleSite/content/_index.md` in 73476bfd6e67dff94b58f61674c44979f450ee9a - discard this commit if you don't want to use it.

- Set the `enableFrontMatter` parameter in `config.toml` to `"true"` to enable the feature
  - If the parameter isn't present, or has a value other than "true", the `<section>` will not be created

- Documentation for the `enableFrontMatter` parameter is available in `exampleSite/content/posts/theme-documentation-basics/`

Hope this is useful. Appreciate any feedback or improvements, as I'm inexperienced with Hugo, and by no means a design buff. I delegate the HTML and CSS prettiness to you, which is why I use this theme to begin with :wink: 

For reference, the upstream Hugo documentation is available here: https://gohugobrasil.netlify.app/templates/homepage/#add-content-and-front-matter-to-the-homepage

### CODE REVIEW

**Comment by yashmehrotra** [comment]
@jamesericdavidson Hi, sorry I was making some changes in the PR and deleted the config.toml by mistake. Can you re-add it. Also, have suggested some changes in the comments. Please have a look.

File: exampleSite/content/posts/theme-documentation-basics.md [hunk 1]
```diff
@@ -162,6 +162,9 @@ In addition to [Hugo global configuration](https://gohugo.io/overview/configurat
 
   # Footer text
   footer = "The Marauders"
+
+  # Add front matter to the homepage, using content/_index.md
+  enableFrontMatter = "true"
```
Comment (yashmehrotra) [inline exampleSite/content/posts/theme-documentation-basics.md:167]: Remove from here

File: exampleSite/content/posts/theme-documentation-basics.md [hunk 2]
```diff
@@ -221,6 +224,15 @@ Text to display in the footer section
   footer = "Text in footer"
 ```
 
+### Enable Front Matter
```
Comment (yashmehrotra) [inline exampleSite/content/posts/theme-documentation-basics.md:227]: Set heading for displaying content at homepage

File: layouts/index.html [hunk 3]
```diff
@@ -25,6 +25,14 @@ <h3>{{ .Site.Params.Description }}</h3>
 </section>
 {{ end }}
 
+{{ if isset .Site.Params "enablefrontmatter" }}
+    {{ if eq .Site.Params.EnableFrontMatter "true" }}
+    <section class="home-content">
```
Comment (yashmehrotra) [inline layouts/index.html:30]: And an `<hr>` tag before section

**Comment by jamesericdavidson** [comment]
Thanks for the review 🙂

I will address your comments later

Regards,
James Davidson

-------- Original Message --------

**Comment by jamesericdavidson** [comment]
Apologies for the delay - internet foibles :smiley:

File: layouts/index.html (hunk 3 above)
Comment (jamesericdavidson) [reply layouts/index.html:31]: 1bcafd01e127cfb2dafcce2813eb73bd5b1a8caa

File: exampleSite/content/posts/theme-documentation-basics.md (hunk 2 above)
Comment (jamesericdavidson) [reply exampleSite/content/posts/theme-documentation-basics.md]: Not sure what you mean, so I reverted the documentation commit in c3d73a65272fc508c88ee814f6c9848f777ea08d

File: exampleSite/content/posts/theme-documentation-basics.md (hunk 1 above)
Comment (jamesericdavidson) [reply exampleSite/content/posts/theme-documentation-basics.md]: 21249b89a93202c56fb58cdc64e97f2580d3ac6d

**Comment by yashmehrotra** [comment]
Thanks for the PR @jamesericdavidson
LGTM, merging

**Comment by jamesericdavidson** [comment]
Thanks for merging :handshake:

**Pull request merged**